rest-client-quickstart/src/main/java/org/csdg8/game/blackjack/BlackjackController.java

```diff
@@ -0,0 +1,86 @@
+package org.csdg8.game.blackjack;
+
+import java.net.URI;
+
+import org.csdg8.game.blackjack.dto.BlackjackActionRequest;
+import org.csdg8.game.blackjack.dto.BlackjackStateResponse;
+import org.csdg8.game.blackjack.dto.GetBlackjackGameResponse;
+import org.csdg8.game.blackjack.dto.GetBlackjackGameResultResponse;
+import org.csdg8.game.blackjack.dto.PlayBlackjackRequest;
+import org.csdg8.model.exception.GameNotFoundException;
+
+import com.google.code.siren4j.component.Entity;
+import com.google.code.siren4j.converter.ReflectingConverter;
+import com.google.code.siren4j.error.Siren4JException;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.ws.rs.core.Response;
+
+@ApplicationScoped
+public class BlackjackController {
+
+    @Inject
+    BlackjackService blackjackService ;
+
+    public Response startGame(@Valid @NotNull PlayBlackjackRequest request) {
+        BlackjackGame game = this.blackjackService.createGame(request.getBetAmount());
+        //here i need to addd a DTO that has the response body of the created game:=)
+        BlackjackStateResponse Blackjack = new BlackjackStateResponse(game.getDealerHand(), game.getPlayerHand(), game.getBetAmount(), game.id, game.getGameResult());
+        return Response.created(URI.create("/game/blackjack/" + game.id)).entity(Blackjack).build();
+    }
+
+    public Entity result(Long id) throws Siren4JException {
+        BlackjackGame game = this.blackjackService.findById(id)
+                .orElseThrow(() -> new GameNotFoundException("No blackjack game found with id %d".formatted(id)));
+
+        GetBlackjackGameResultResponse gameResultResponse = new GetBlackjackGameResultResponse(game.id, game.getChoice(), game.getResult(),
+                game.getBetAmount(), game.getGameResult());
+        return ReflectingConverter.newInstance().toEntity(gameResultResponse);
+    }
+
+    public Entity game() throws Siren4JException {
+        GetBlackjackGameResponse gameResponse = new GetBlackjackGameResponse();
+        return ReflectingConverter.newInstance().toEntity(gameResponse);
+    }
+
+    public Response performAction(long gameId, @Valid @NotNull BlackjackActionRequest request) {
+        // Fetch the game by its ID
+        BlackjackGame game = this.blackjackService.findById(gameId)
+            .orElseThrow(() -> new GameNotFoundException("No blackjack game found with id %d".formatted(gameId)));
```

Comment:
Haven't tested it myself, but what happens if i perform an action on a finished game?